lib/src/models/user.dart

```diff
@@ -11,6 +11,7 @@ class User {
   final List<Profile> profiles;
   final String? referralLink;
   final bool tosAccepted;
+  final String? aiDailyMessageLimit;
```

Comment:
Thinking about this more, how about we make this an `int` instead of a `String?`? We can do the conversion here so apps don't have to deal with the `null` and `NaN` scenarios themselves.

Maybe something like:
```dart
int _getAIDailyMessageLimitProperty(Map<String, dynamic> json) =>
    int.tryParse(json['aiDailyMessageLimit']?.first?['value'] ?? '0') ?? 0;
```